<?xml version="1.0" encoding="UTF-8" ?>
<testsuite tests="29" failures="13" name="TestSuite" time="0.052" errors="0" skipped="0">
  <properties>
    <property name="idea.version" value="2021.3.2"/>
    <property name="java.runtime.name" value="OpenJDK Runtime Environment"/>
    <property name="java.vm.version" value="17.0.8.1+8-LTS"/>
    <property name="sun.boot.library.path" value="C:\Users\Alexander\.jdks\corretto-17.0.8.1\bin"/>
    <property name="maven.multiModuleProjectDirectory" value="E:\RNTgroup\AT\AT_Module_1"/>
    <property name="java.vm.vendor" value="Amazon.com Inc."/>
    <property name="java.vendor.url" value="https://aws.amazon.com/corretto/"/>
    <property name="guice.disable.misplaced.annotation.check" value="true"/>
    <property name="path.separator" value=";"/>
    <property name="java.vm.name" value="OpenJDK 64-Bit Server VM"/>
    <property name="sun.os.patch.level" value=""/>
    <property name="user.script" value=""/>
    <property name="user.country" value="RU"/>
    <property name="sun.java.launcher" value="SUN_STANDARD"/>
    <property name="java.vm.specification.name" value="Java Virtual Machine Specification"/>
    <property name="user.dir" value="E:\RNTgroup\AT\AT_Module_1"/>
    <property name="java.vm.compressedOopsMode" value="32-bit"/>
    <property name="java.runtime.version" value="17.0.8.1+8-LTS"/>
    <property name="os.arch" value="amd64"/>
    <property name="java.io.tmpdir" value="C:\Users\ALEXAN~1\AppData\Local\Temp\"/>
    <property name="line.separator" value="
"/>
    <property name="java.vm.specification.vendor" value="Oracle Corporation"/>
    <property name="user.variant" value=""/>
    <property name="os.name" value="Windows 10"/>
    <property name="maven.ext.class.path" value="C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2021.3.2\plugins\maven\lib\maven-event-listener.jar"/>
    <property name="classworlds.conf" value="C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2021.3.2\plugins\maven\lib\maven3\bin\m2.conf"/>
    <property name="sun.jnu.encoding" value="Cp1251"/>
    <property name="java.library.path" value="C:\Users\Alexander\.jdks\corretto-17.0.8.1\bin;C:\WINDOWS\Sun\Java\bin;C:\WINDOWS\system32;C:\WINDOWS;C:\Program Files\Common Files\Oracle\Java\javapath;C:\Program Files (x86)\Common Files\Intel\Shared Libraries\redist\intel64\compiler;C:\Program Files (x86)\Intel\iCLS Client\;C:\Program Files\Intel\iCLS Client\;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\DAL;C:\Program Files\Intel\Intel(R) Management Engine Components\DAL;C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\IPT;C:\Program Files\Intel\Intel(R) Management Engine Components\IPT;C:\Program Files\Intel\WiFi\bin\;C:\Program Files\Common Files\Intel\WirelessCommon\;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files\NVIDIA Corporation\NVIDIA NvDLISR;C:\Program Files (x86)\Brackets\command;C:\Program Files\nodejs\;C:\chromedriver;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\110\Tools\Binn\;C:\Program Files\dotnet\;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files\Azure Data Studio\bin;C:\Program Files (x86)\Microsoft SQL Server\160\Tools\Binn\;C:\Program Files\Microsoft SQL Server\160\Tools\Binn\;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\170\Tools\Binn\;C:\Program Files\Microsoft SQL Server\160\DTS\Binn\;C:\Program Files (x86)\Microsoft SQL Server\160\DTS\Binn\;C:\Program Files\Git\cmd;C:\Users\Alexander\AppData\Local\Microsoft\WindowsApps;C:\Program Files\Intel\WiFi\bin\;C:\Program Files\Common Files\Intel\WirelessCommon\;C:\Users\Alexander\AppData\Roaming\npm;C:\Users\Alexander\.dotnet\tools;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2021.3.2\bin;;C:\Users\Alexander\AppData\Local\Programs\Microsoft VS Code\bin;C:\Program Files\Azure Data Studio\bin;C:\Program Files\apache-maven-3.9.6\bin;;."/>
    <property name="maven.conf" value="C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2021.3.2\plugins\maven\lib\maven3/conf"/>
    <property name="jdk.debug" value="release"/>
    <property name="java.class.version" value="61.0"/>
    <property name="java.specification.name" value="Java Platform API Specification"/>
    <property name="sun.management.compiler" value="HotSpot 64-Bit Tiered Compilers"/>
    <property name="os.version" value="10.0"/>
    <property name="user.home" value="C:\Users\Alexander"/>
    <property name="user.timezone" value="Europe/Moscow"/>
    <property name="file.encoding" value="UTF-8"/>
    <property name="java.specification.version" value="17"/>
    <property name="user.name" value="Alexander"/>
    <property name="java.class.path" value="C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2021.3.2\plugins\maven\lib\maven3\boot\plexus-classworlds-2.6.0.jar;C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2021.3.2\plugins\maven\lib\maven3\boot\plexus-classworlds.license"/>
    <property name="java.vm.specification.version" value="17"/>
    <property name="sun.arch.data.model" value="64"/>
    <property name="sun.java.command" value="org.codehaus.classworlds.Launcher -Didea.version=2021.3.2 package"/>
    <property name="java.home" value="C:\Users\Alexander\.jdks\corretto-17.0.8.1"/>
    <property name="user.language" value="ru"/>
    <property name="java.specification.vendor" value="Oracle Corporation"/>
    <property name="java.vm.info" value="mixed mode, sharing"/>
    <property name="java.version" value="17.0.8.1"/>
    <property name="native.encoding" value="Cp1251"/>
    <property name="java.vendor" value="Amazon.com Inc."/>
    <property name="maven.home" value="C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2021.3.2\plugins\maven\lib\maven3"/>
    <property name="file.separator" value="\"/>
    <property name="java.version.date" value="2023-08-22"/>
    <property name="java.vendor.url.bug" value="https://github.com/corretto/corretto-17/issues/"/>
    <property name="sun.io.unicode.encoding" value="UnicodeLittle"/>
    <property name="sun.cpu.endian" value="little"/>
    <property name="java.vendor.version" value="Corretto-17.0.8.8.1"/>
    <property name="sun.cpu.isalist" value="amd64"/>
  </properties>
  <testcase classname="org.rntgroup.test.PowerFunctionTest" name="unitPowTest" time="0.008"/>
  <testcase classname="org.rntgroup.test.PowerFunctionTest" name="unitPowTest" time="0.001"/>
  <testcase classname="org.rntgroup.test.PowerFunctionTest" name="unitPowTest" time="0.018">
    <failure message="Result is not correct! expected [46.7653718] but found [27.0]" type="java.lang.AssertionError">java.lang.AssertionError: Result is not correct! expected [46.7653718] but found [27.0]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEquals(Assert.java:728)
	at org.rntgroup.test.PowerFunctionTest.unitPowTest(PowerFunctionTest.java:15)
</failure>
  </testcase>
  <testcase classname="org.rntgroup.test.PowerFunctionTest" name="unitPowTest" time="0"/>
  <testcase classname="org.rntgroup.test.PowerFunctionTest" name="unitSqrtNegativeNumbersTest" time="0.003">
    <failure message="Expected exception not throw" type="java.lang.AssertionError">java.lang.AssertionError: Expected exception not throw
	at org.testng.Assert.fail(Assert.java:111)
	at org.rntgroup.test.PowerFunctionTest.unitSqrtNegativeNumbersTest(PowerFunctionTest.java:30)
</failure>
  </testcase>
  <testcase classname="org.rntgroup.test.PowerFunctionTest" name="unitSqrtTest" time="0"/>
  <testcase classname="org.rntgroup.test.PowerFunctionTest" name="unitSqrtTest" time="0.001"/>
  <testcase classname="org.rntgroup.test.PowerFunctionTest" name="unitSqrtTest" time="0.001"/>
  <testcase classname="org.rntgroup.test.SingDefinitionTest" name="unitIsNegativeTest" time="0"/>
  <testcase classname="org.rntgroup.test.SingDefinitionTest" name="unitIsNegativeTest" time="0.001"/>
  <testcase classname="org.rntgroup.test.SingDefinitionTest" name="unitIsNegativeTest" time="0.001"/>
  <testcase classname="org.rntgroup.test.SingDefinitionTest" name="unitIsPositiveTest" time="0.001"/>
  <testcase classname="org.rntgroup.test.SingDefinitionTest" name="unitIsPositiveTest" time="0.001"/>
  <testcase classname="org.rntgroup.test.SingDefinitionTest" name="unitIsPositiveTest" time="0.001"/>
  <testcase classname="org.rntgroup.test.TrigonometricFunctionTest" name="unitCosTest" time="0.003">
    <failure message="Result is not correct! expected [0.5] but found [0.8660254037844386]" type="java.lang.AssertionError">java.lang.AssertionError: Result is not correct! expected [0.5] but found [0.8660254037844386]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEquals(Assert.java:728)
	at org.rntgroup.test.TrigonometricFunctionTest.unitCosTest(TrigonometricFunctionTest.java:23)
</failure>
  </testcase>
  <testcase classname="org.rntgroup.test.TrigonometricFunctionTest" name="unitCosTest" time="0.002">
    <failure message="Result is not correct! expected [-1.0] but found [-1.2246467991473532E-16]" type="java.lang.AssertionError">java.lang.AssertionError: Result is not correct! expected [-1.0] but found [-1.2246467991473532E-16]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEquals(Assert.java:728)
	at org.rntgroup.test.TrigonometricFunctionTest.unitCosTest(TrigonometricFunctionTest.java:23)
</failure>
  </testcase>
  <testcase classname="org.rntgroup.test.TrigonometricFunctionTest" name="unitCosTest" time="0.002">
    <failure message="Result is not correct! expected [1.0] but found [0.0]" type="java.lang.AssertionError">java.lang.AssertionError: Result is not correct! expected [1.0] but found [0.0]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEquals(Assert.java:728)
	at org.rntgroup.test.TrigonometricFunctionTest.unitCosTest(TrigonometricFunctionTest.java:23)
</failure>
  </testcase>
  <testcase classname="org.rntgroup.test.TrigonometricFunctionTest" name="unitCtgTest" time="0.001">
    <failure message="Result is not correct! expected [0.0] but found [0.9171523356672744]" type="java.lang.AssertionError">java.lang.AssertionError: Result is not correct! expected [0.0] but found [0.9171523356672744]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEquals(Assert.java:728)
	at org.rntgroup.test.TrigonometricFunctionTest.unitCtgTest(TrigonometricFunctionTest.java:39)
</failure>
  </testcase>
  <testcase classname="org.rntgroup.test.TrigonometricFunctionTest" name="unitCtgTest" time="0.002">
    <failure message="Result is not correct! expected [0.404] but found [0.9606886408725873]" type="java.lang.AssertionError">java.lang.AssertionError: Result is not correct! expected [0.404] but found [0.9606886408725873]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEquals(Assert.java:728)
	at org.rntgroup.test.TrigonometricFunctionTest.unitCtgTest(TrigonometricFunctionTest.java:39)
</failure>
  </testcase>
  <testcase classname="org.rntgroup.test.TrigonometricFunctionTest" name="unitCtgTest" time="0.001">
    <failure message="Result is not correct! expected [-1.0] but found [-0.6557942026326724]" type="java.lang.AssertionError">java.lang.AssertionError: Result is not correct! expected [-1.0] but found [-0.6557942026326724]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEquals(Assert.java:728)
	at org.rntgroup.test.TrigonometricFunctionTest.unitCtgTest(TrigonometricFunctionTest.java:39)
</failure>
  </testcase>
  <testcase classname="org.rntgroup.test.TrigonometricFunctionTest" name="unitSinTest" time="0"/>
  <testcase classname="org.rntgroup.test.TrigonometricFunctionTest" name="unitSinTest" time="0">
    <failure message="Result is not correct! expected [0.5] but found [0.49999999999999994]" type="java.lang.AssertionError">java.lang.AssertionError: Result is not correct! expected [0.5] but found [0.49999999999999994]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEquals(Assert.java:728)
	at org.rntgroup.test.TrigonometricFunctionTest.unitSinTest(TrigonometricFunctionTest.java:15)
</failure>
  </testcase>
  <testcase classname="org.rntgroup.test.TrigonometricFunctionTest" name="unitSinTest" time="0"/>
  <testcase classname="org.rntgroup.test.TrigonometricFunctionTest" name="unitSinTest" time="0"/>
  <testcase classname="org.rntgroup.test.TrigonometricFunctionTest" name="unitTgTest" time="0"/>
  <testcase classname="org.rntgroup.test.TrigonometricFunctionTest" name="unitTgTest" time="0.001">
    <failure message="Result is not correct! expected [0.0] but found [1.0]" type="java.lang.AssertionError">java.lang.AssertionError: Result is not correct! expected [0.0] but found [1.0]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEquals(Assert.java:728)
	at org.rntgroup.test.TrigonometricFunctionTest.unitTgTest(TrigonometricFunctionTest.java:31)
</failure>
  </testcase>
  <testcase classname="org.rntgroup.test.TrigonometricFunctionTest" name="unitTgTest" time="0.002">
    <failure message="Result is not correct! expected [0.364] but found [1.0]" type="java.lang.AssertionError">java.lang.AssertionError: Result is not correct! expected [0.364] but found [1.0]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEquals(Assert.java:728)
	at org.rntgroup.test.TrigonometricFunctionTest.unitTgTest(TrigonometricFunctionTest.java:31)
</failure>
  </testcase>
  <testcase classname="org.rntgroup.test.TrigonometricFunctionTest" name="unitTgTest" time="0.001">
    <failure message="Result is not correct! expected [0.0] but found [1.0]" type="java.lang.AssertionError">java.lang.AssertionError: Result is not correct! expected [0.0] but found [1.0]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEquals(Assert.java:728)
	at org.rntgroup.test.TrigonometricFunctionTest.unitTgTest(TrigonometricFunctionTest.java:31)
</failure>
  </testcase>
  <testcase classname="org.rntgroup.test.TrigonometricFunctionTest" name="unitTgTest" time="0">
    <failure message="Result is not correct! expected [-1.0] but found [1.0]" type="java.lang.AssertionError">java.lang.AssertionError: Result is not correct! expected [-1.0] but found [1.0]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEquals(Assert.java:728)
	at org.rntgroup.test.TrigonometricFunctionTest.unitTgTest(TrigonometricFunctionTest.java:31)
</failure>
  </testcase>
</testsuite>